src/test/java/com/reservation/payment/domain/PaymentTestBuilder.java

```diff
@@ -0,0 +1,50 @@
+package com.reservation.payment.domain;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+public class PaymentTestBuilder {
```

Comment:
```suggestion
class PaymentTestBuilder {
```

cała ta klasa (i ofc też wszystkie jej metody) może być package-private.